queries/rbs/highlights.scm

```diff
@@ -0,0 +1,42 @@
+["(" ")" "{" "}" "[" "]"] @punctuation.brackets
+["," "."]                 @punctuation.delimiter
+(method_signatures "|"    @punctuation.special)
+
+["class" "interface" "module" "end" "def" "alias"] @keyword.scope
+
+["=" "->" "<" "::"]           @operator
+(supertype)                   @operator
+(union_type "|")              @operator
+(intersection_type "&")       @operator
+[":" "?"]                     @operator.special
+(splat_parameter "*")         @operator.special
+(double_splat_parameter "**") @operator.special
+
+["type" (visibility) (unchecked) (variance)] @keyword
+
+["include" "extend" "prepend"] @include
+
+[(class_name) (constant_name) (module_name) (interface_name)] @type.constant
```

Comment:
`(constant_name)` should be `@constant`.
`(module_name)` should be `@namespace`?
`(class_name)` and `(interface_name)` should be `@type`.